src/traccuracy/_tracking_graph.py

```diff
@@ -448,7 +314,7 @@ def get_preds(self, node):
         """
         return [pred for pred, _ in self.graph.in_edges(node)]
 
-    def get_succs(self, node):
+    def get_succs(self, node: Hashable) -> list[Hashable]:
```

Comment:
We should revisit our re-implementation of get_preds and get_succs @msschwartz21. I think we didn't know we were exactly re-implementing [Graph.successors](https://github.com/networkx/networkx/blob/862b0e95d2008c7e032fbdb907168927c24b4255/networkx/classes/digraph.py#L874) since we use `out_edges` in our implementation - at the very least we should use the more appropriate networkx function. Is this still worth wrapping, so that we don't need to call `tracking_graph.graph.successors(n)`? Should we mimic the function name?

Reply:
Good catch! Let's just drop the functions entirely. I think we really only use them in the division metrics and it doesn't add that much to just use the `networkx` api in this case.